jobs/build/custom/Jenkinsfile

```diff
@@ -91,6 +91,8 @@ node {
     buildlib.initialize(false)
     GITHUB_BASE = "git@github.com:openshift" // buildlib uses this global var
 
+    majorVersion = params.BUILD_VERSION.split('.')[0]
+    minorVersion = params.BUILD_VERSION.split('.')[1]
```

Comment:
We should really standardize this in commonlib somehow. But for now, whatever.

Reply:
I agree.